modules/fedimint-lnv2-client/src/cli.rs

```diff
@@ -4,21 +4,23 @@ use clap::Parser;
 use fedimint_api_client::api::FederationApiExt;
 use fedimint_core::module::ApiRequestErased;
 use fedimint_core::util::SafeUrl;
+use fedimint_core::PeerId;
 use fedimint_lnv2_common::endpoint_constants::{
     ADD_GATEWAY_ENDPOINT, GATEWAYS_ENDPOINT, REMOVE_GATEWAY_ENDPOINT,
 };
 use serde::Serialize;
 
+use crate::api::LnFederationApi;
 use crate::LightningClientModule;
 
 #[derive(Parser, Serialize)]
 enum Opts {
+    /// Fetch vetted gateways
+    Gateways { peer: Option<PeerId> },
```

Comment:
`ListGateways` to match?

Or you could have a `Gateway` subcommand that has `Add`, `List`, `Remove` command.

Reply:
I choose gateways, because its shorter, the only user command of the three and more canonical with how I named the endpoints; list has relatively little meaning like "fetch".

Reply:
@joschisan I don't know. Seems kind of inconsistent.

BTW. We could have aliases to make common commands more convenient to type.